k8s/pink-time/pinkconf.yaml

```diff
@@ -0,0 +1,8 @@
+pinkconf:
+  bestcolor:
+    - pink
+  name: molberte
+  lore:
+    - "ya"
+    - "ochen"
+    - "ustal"
```

Comment:
this lore.... is so deep.....
![](https://user-images.githubusercontent.com/49134679/205136365-f000312d-b021-4696-b9f2-d7f329895792.png)
(and so ckufin true)